sea-orm-codegen/src/entity/writer.rs

```diff
@@ -594,8 +615,15 @@ impl EntityWriter {
                         }
                         ts = quote! { #ts #attr };
                     }
-                    quote! {
-                        #[sea_orm(#ts)]
+                    if skip_pk_deserialization {
+                        quote! {
+                            #[sea_orm(#ts)]
+                            #[serde(skip_deserialization)]
+                        }
+                    } else {
+                        quote! {
+                            #[sea_orm(#ts)]
+                        }
```

Comment:
Hey @Witcher01, sorry for the delay! Thanks for the great contributions!! Looks good overall :)

I have a minor nitpick. Please check if the refactoring make any sense
- https://github.com/Witcher01/sea-orm/compare/skip-serde-deserialization...billy1624:pr/846

Reply:
No worries! I'm not sure what you mean, my commit looks good to me, tests run, and running it gives me the right results, too.
Could you elaborate? Thanks.

Reply:
I don't want to keep a mutable `skip_pk_deserialization` and that's why refactored the code :P